packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { User } from '@microsoft/microsoft-graph-types';
-import { customElement, html, internalProperty, property, TemplateResult } from 'lit-element';
```

Comment:
`state` is a lit-element v3 concept, `internalProperty` is the v2 version, does this work?

Reply:
Yes. In v2, `internalProperty` is deprecated for `state` which is made available from `lit-element`. Then in v3, `state` is moved to `lit/decorators.js`. Therefore, when the merge to next is done, this should be updated to reflect the import is from `lit/decorators.js`